config/checkstyle_non_main_files_checks.xml

```diff
@@ -19,7 +19,7 @@
     <module name="RegexpSingleline">
       <!-- catch lines above 100 symbols -->
       <property name="format"
-                value="^(?!(.*href|.*http|import|.* com.*|.*com/.*|.* files=)).{101,}$"/>
+                value="^(?!(.*href|.*http|import|.* com\..*|.*com\..*|.* files=|.*\.dtd)).{101,}$"/>
```

Comment:
What is the original purpose of adding the `com` ones? From what I saw, it was for the XML files and their lines with multiple packages. Does it make sense we change the `.*` before them to something that will more specifically target the XML files instead of hitting our inputs by accident?

Reply:
fully qualified names.
We need to change this to wrap lines, most Inputs do not need this, it is just leftover. I created separate issue on this, to speedup validation enforcement ASAP.